com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/flowable/RetryProcessAction.java

```diff
@@ -30,20 +30,16 @@ public RetryProcessAction(FlowableFacade flowableFacade, List<AdditionalProcessA
 
     @Override
     protected void executeActualProcessAction(String user, String superProcessInstanceId) {
-        List<String> subProcessIds = getActiveExecutionIds(superProcessInstanceId);
-        ListIterator<String> subProcessesIdsIterator = subProcessIds.listIterator(subProcessIds.size());
-
+        List<String> inactiveExecutionIdsWithoutChildren = flowableFacade.getInactiveExecutionIdsWithoutChildren(superProcessInstanceId);
```

Comment:
I think that the inactive executions will be removed from the execution table when some time passes. That is why all the active executions are taken. Is there any particular reason why this is changed?

Reply:
Reverted as it was before. The main idea was to take only executions.